packages/plugins/rpc-manager/src/lib/eth_unsubscribe.ts

```diff
@@ -0,0 +1,47 @@
+import { Callback, Embark, Events } /* supplied by @types/embark in packages/embark-typings */ from "embark";
+import Web3RequestManager from "web3-core-requestmanager";
+import { __ } from "embark-i18n";
+import Web3 from "web3";
+import RpcModifier from "./rpcModifier";
+
+export default class EthUnsubscribe extends RpcModifier {
```

Comment:
Are you referring to the `Web3` and `Web3RequestManager` imports? If so, they are unused, so should be removed from the `EthUnsubscribe` class.

Reply:
It was more in regards to making it more DRY, since `EthUnsubscribe` and `EthSubscribe` share a lot in common

Reply:
Ah ok, I see what you mean. We could combine them both in to one `RPCModifier`. However, I feel like it might make things a bit harder to read as it would include conditions for subscribe/unsubscribe. Either way would work. Happy to do it if you think it's necessary.

I'm keen to abstract a bit more in to the `RPCModifier` class so there is automatic logging and less setup involved, which should make things a bit more DRY.